Makefile

```diff
@@ -122,7 +122,10 @@ $(GO) install -v -gcflags "-N -l" $(BUILD_GOPATH)/src/github.com/openconfig/ygot
         cd $(BUILD_GOPATH)/src/github.com/openconfig/goyang/; git reset --hard HEAD; git checkout 064f9690516f4f72db189f4690b84622c13b7296 >/dev/null ; true; \
 	cp $(TOPDIR)/goyang-modified-files/goyang.patch .; \
 	patch -p1 < goyang.patch; rm -f goyang.patch; \
-	$(GO) install -v -gcflags "-N -l" $(BUILD_GOPATH)/src/github.com/openconfig/goyang
+	$(GO) install -v -gcflags "-N -l" $(BUILD_GOPATH)/src/github.com/openconfig/goyang; \
+	cd $(BUILD_GOPATH)/src/github.com/antchfx/jsonquery; git reset --hard HEAD; \
+	git checkout 3535127d6ca5885dbf650204eb08eabf8374a274 2>/dev/null ; \
```

Comment:
Statements like "git checkout 3535127d6ca5885dbf650204eb08eabf8374a274" is already present in another target **go-pkg-version**. Will there be a race?

Reply:
Kept only one checkout under go-patch which is anyway included in 'all' rule.